float64[] joint_goal
---
bool answer